### DISCUSSION TITLE
Insert an alarm list into the view

### DISCUSSION THREAD

**Discussion by nex2000** [frangoteam/FUXA#1384] [Q&A]
I would like to insert a datagrid with the list of alarms in a view, is it possible?

Regards

**Comment by unocelli** [comment]
Hi, No currently is not possible. Should be a extension of the table control
#1310

**Comment by robsori** [comment]
Yes, this feature would be very useful. An extension of the table control but to contain alarms. The control should allow the configuration of a filter for a specific mechanism so that alarms are displayed only for this mechanism.
